thesis/behaviour.som

```diff
@@ -0,0 +1,45 @@
+class Behaviour usingPlatform: platform = Value (
```

Comment:
Are the following files really to be committed here? seems like something you'd like in a separate repo.
Considering that the folder is named `thesis`?